Develop/scripts/script.js

```diff
@@ -0,0 +1,198 @@
+$(document).ready(function () {
```

Comment:
Do they require you to write your application using jQuery? Most companies you apply with will require vanilla JS knowledge or modern single page application framework knowledge like React, Vue, Angular etc. I'd suggest you ask them if you can do it just with Vanilla JS next week. Good to learn :)